content/docs/architecture/step-registry.md

```diff
@@ -166,6 +166,40 @@ ref:
 Note that access to read these secrets from the namespace configured must be granted separately from the configuration being added to a step.
 By default, only secrets in the `test-credentials` namespace will be available for mounting into test steps.
 
+#### Injecting the `oc` CLI
+
+Steps can make the `oc` CLI available to their commands by adding the `cli` configuration item to the test step, specifying which OpenShift
+release the CLI should be sourced from. For example, the following configuration pulls in a CentOS image, a custom OCP release and runs a test
+where the release's CLI is injected to the test step.
+
+{{< highlight yaml >}}
+base_images:
+  os: # import an image and call it "os"
+    name: centos
+    namespace: openshift
+    tag: '7'
+releases:
+  custom: # import a release and call it "custom"
+    candidate:
+      product: okd
+      version: "4.3"
+tests:
+  - as: with-cli
+    steps:
+      test:
+        - as: use-cli
+          commands: oc adm policy add-role-to-user --help
+          from: os    # use the "os" image for running the test
+          cli: custom # allow the CLI from the "custom" release to be available
```

Comment:
Wouldn't it make more sense to use an actual release like "4.6" to clarify that this has to be a release version?

Reply:
That is not what this field should be, so no.

Reply:
Then I don't understand it. Isn't the purpose of this to inject a `oc` cli from a specifig release?

Reply:
The release is called `"custom"`, as noted in the `releases` configuration above.

Reply:
https://github.com/openshift/ci-docs/pull/8/files#diff-eb2602965a2ae9f2a413ef8521d02cd1ca27fa8cb292d7035e9ea86c30a951d6R182